docs/security-considerations.rst

```diff
@@ -88,8 +88,7 @@ outlined further below:
       function withdraw() {
           var share = shares[msg.sender];
           shares[msg.sender] = 0;
-          if (!msg.sender.send(share))
-              throw;
+          require(msg.sender.send(share));
```

Comment:
This is the withdraw pattern, in here probably we should prefer `transfer()` instead.